es/util.go

```diff
@@ -136,6 +155,11 @@ func normalizedIndexLifecyclePolicy(policy map[string]interface{}) map[string]in
 	f := flattenMap(policy)
 	for k, v := range f {
 		f[k] = fmt.Sprintf("%v", v)
+		// Supress phases.delete.actions.delete.delete_searchable_snapshot = true that is included by default
+		// starting in 7.8
+		if k == "phases.delete.actions.delete.delete_searchable_snapshot" && fmt.Sprintf("%v", v) == "true" {
```

Comment:
Hm, will this potentially also ignore it if it's set to true by the user? I think the better approach for these is to ignore these in the diff if they come back from ES but are not in terraform state.

Reply:
added a commit to back this out for now, in order to merge the rest of the PR!